src/openloco/companymgr.cpp

```diff
@@ -140,6 +140,21 @@ namespace openloco::companymgr
         return regs.bx;
     }
 
+    owner_status getOwnerStatus(company_id_t id)
+    {
+        registers regs;
+        regs.esi = (int32_t)get(id);
+        call(0x00438047, regs);
+
+        owner_status ownerStatus;
+
+        ownerStatus.string = regs.bx;
+        ownerStatus.argument1 = regs.ecx;
+        ownerStatus.argument2 = regs.edx;
+
+        return ownerStatus;
```

Comment:
This variable is on the stack, and goes out of scope when this function returns.

Reply:
Should I pass in a pointer to an `owner_status` variable and not return a value instead?

Reply:
It's fine it's sending back a copy (probably is actually a move not a copy but that's besides the point) not a pointer to the stack.